src/objects.js

```diff
@@ -1,41 +1,67 @@
+const Person = class {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+};
+
 const createPerson = (name, age) => {
   // your code here
+  return new Person(name, age);
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  return Object.hasOwn(object, property);
 };
 
 const isOver65 = person => {
   // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
   // your code here
+  return people.map(x => x.age);
```

Comment:
As before, it would be much better to use a better variablle name in place of `x`